.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: Release
+
+on:
+  push:
+    branches: [master]
```

Comment:
releases are usually on separate branches, how it'll work with them?

Reply:
Tbh I did not consider that. The branch restriction should probably be removed.